src/launchpad/analyzers/apple.py

```diff
@@ -149,6 +150,19 @@ def _extract_app_info(self, xcarchive: ZippedXCArchive) -> AppleAppInfo:
 
         supported_platforms = plist.get("CFBundleSupportedPlatforms", [])
         is_simulator = "iphonesimulator" in supported_platforms or plist.get("DTPlatformName") == "iphonesimulator"
+
+        # Validate code signature
+        is_valid = True
+        errors: List[str] = []
+        try:
+            validator = CodeSignatureValidator(xcarchive)
+            is_valid, errors = validator.validate()
+        except Exception as e:
+            logger.warning(f"Failed to validate code signature: {e}")
+            is_valid = False
+            errors = [str(e)]
+
+        logger.info(f"Code signature validation errors: {errors}")
```

Comment:
Can this be moved into the error block? Don't think we need to see its output for the happy path.